utils/url_generator.py

```diff
@@ -0,0 +1,227 @@
+from typing import List
```

Comment:
To be more explicit, call this file linkedin_url_generator.py as it currently generates link only for Linkedin